.github/workflows/vdscanner_tools_delivery.yml

```diff
@@ -0,0 +1,138 @@
+
+jobs:
+  build:
+    name: VD Scanner Tools Delivery
+
+    # Runs only if the PR status is different to Draft
+    if: ${{ !github.event.pull_request.draft }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 60
+
+    steps:
+      - name: Install dependencies
+        run: sudo apt-get install lzip xz-utils
+
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Project dependencies
+        uses: ./.github/actions/vulnerability_scanner_deps
+        with:
+          test: "false"
+
+      - name: Router
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/router
+          test: "false"
+
+      - name: Indexer connector
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/indexer_connector
+          test: "false"
+
+      - name: Content manager
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/shared_modules/content_manager
+          test: "false"
+
+      - name: Vulnerability scanner
+        uses: ./.github/actions/compile_and_test
+        with:
+          path: src/wazuh_modules/vulnerability_scanner
+          test: "false"
+
+      - name: Prepare vdscanner tools
+        run: |
+          # Paths
+          SRC_DIR=${{github.workspace}}/src/
+          SHARED_MODULES_DIR=${{github.workspace}}/src/shared_modules/
+          EXTERNAL_DIR=${{github.workspace}}/src/external/
+          VDSCANNER_DIR=${{github.workspace}}/src/wazuh_modules/vulnerability_scanner/
+
+          # Copy flatbuffer schemas
+          cp -r ${VDSCANNER_DIR}schemas ${{env.TOOLS_DIR}}
+          cp ${SHARED_MODULES_DIR}utils/flatbuffers/schemas/syscollectorDeltas/syscollector_deltas.fbs ${{env.TOOLS_DIR}}/schemas
+          cp ${SHARED_MODULES_DIR}utils/flatbuffers/schemas/syscollectorRsync/syscollector_synchronization.fbs ${{env.TOOLS_DIR}}/schemas
+
+          # Copy required libraries
+          cp ${SRC_DIR}libwazuhext.so ${{env.TOOLS_DIR}}
+          cp ${EXTERNAL_DIR}rocksdb/build/librocksdb.so.8.3.2 ${{env.TOOLS_DIR}}librocksdb.so.8
+          cp ${SHARED_MODULES_DIR}indexer_connector/build/libindexer_connector.so ${{env.TOOLS_DIR}}
+          cp ${SHARED_MODULES_DIR}router/build/librouter.so ${{env.TOOLS_DIR}}
+          cp ${SHARED_MODULES_DIR}content_manager/build/libcontent_manager.so ${{env.TOOLS_DIR}}
+          cp ${VDSCANNER_DIR}build/libvulnerability_scanner.so ${{env.TOOLS_DIR}}
+
+          # Copy configuration and mocked data files
+          cp ${VDSCANNER_DIR}indexer/template/index-template.json ${{env.TOOLS_DIR}}
+          cp ${VDSCANNER_DIR}testtool/scanner/config.json ${{env.TOOLS_DIR}}
+          cp ${VDSCANNER_DIR}testtool/scanner/fakeAgentData/agentHotfixesData.json ${{env.TOOLS_DIR}}
+          cp ${VDSCANNER_DIR}testtool/scanner/fakeGlobalData/globalData.json ${{env.TOOLS_DIR}}
+
```

Comment:
This makes me think that it's more correct that the other repository has it's own configuration files.
What do you think?